sql_generators/glean_usage/templates/baseline_clients_daily_v1.backfill.yaml

```diff
@@ -0,0 +1,10 @@
+{{ header_yaml }}
+{% if app_name == "fenix" -%}
```

Comment:
This is to make sure if we ever add more backfill configuration we do not accidentally add this entry to other app ids.